.github/workflows/00_pull_request_entry_point.yml

```diff
@@ -0,0 +1,50 @@
+name: Pull Request Entry Point
+
+on:
+  pull_request:
+
+jobs:
+  check-permissions:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'run-no-ci') }}
+    uses: ./.github/workflows/90_helpers_check-permissions.yml
```

Comment:
these reusable workflows should *probably* move into something like https://github.com/neondatabase/github-workflows in the long run, so that we can reuse them from all sorts of different repos, right?

Reply:
yes, it's the final goal. just having them there now helps with trying them, and when they will be more ready, we should move them into separate repo and apply to the all or most of our repos